tests/ctl/core/test_system.py

```diff
@@ -7,44 +7,47 @@
 from py._path.local import LocalPath
 
 from fides.ctl.connectors.models import OktaConfig
-from fides.ctl.core import api
 from fides.ctl.core import system as _system
+from fides.ctl.core.api import generate_resource_url
 from fides.ctl.core.config import FidesConfig
 
 
-def create_server_systems(test_config: FidesConfig, systems: List[System]) -> None:
+def create_server_systems(
+    test_config: FidesConfig, systems: List[System], test_client
+) -> None:
     for system in systems:
-        api.create(
-            url=test_config.cli.server_url,
-            resource_type="system",
-            json_resource=system.json(exclude_none=True),
+        test_client.post(
+            generate_resource_url(test_config.cli.server_url, "system"),
             headers=test_config.user.request_headers,
+            data=system.json(exclude_none=True),
         )
 
 
-def delete_server_systems(test_config: FidesConfig, systems: List[System]) -> None:
+def delete_server_systems(
+    test_config: FidesConfig, systems: List[System], test_client
+) -> None:
     for system in systems:
-        api.delete(
-            url=test_config.cli.server_url,
-            resource_type="system",
-            resource_id=system.fides_key,
+        test_client.delete(
+            generate_resource_url(
+                test_config.cli.server_url, "system", resource_id=system.fides_key
+            ),
             headers=test_config.user.request_headers,
         )
 
 
 @pytest.fixture(scope="function")
 def create_test_server_systems(
```

Comment:
This fixture is now using the `test_client` directly which is aware of test_mode, instead of using the `api.create`/`api.delete` methods so I believe this fixture is creating the resources on the test db.  

Unfortunately I think methods outside of the ctl API (which are injecting sessions) are still talking to the application db, and I'm not sure how to resolve yet - So `test_get_all_server_systems` is failing but I think that it's looking for the resources in the application db.